<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetVotesItem</class>
 <widget class="QWidget" name="WidgetVotesItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  color: white;
}
QGroupBox {
  border-radius: 0px;
  border: 0px solid rgb(131, 147, 167);
  background-color: transparent; /*rgb(80,112,125);*/
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QTextEdit {
  background-color: rgb(101,117, 137);  
}
QPlainTextEdit {
  background-color: rgb(101,117, 137);  
}
QComboBox {
  background-color: rgb(101,117, 137);  
}
QCheckBox::indicator {
  border: 2px solid rgb(80,112,125);
}
QCheckBox::indicator:checked {
  background-color: white;
}
QListView {
  background-color: rgb(101,117, 137);  
}
QTableWidget {
  background-color: rgb(101,117, 137);  
}
QDateTimeEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background: rgb(101,117, 137);
}
QPushButton::checked {
  border: 2px solid white;
  font-weight: bold;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBoxHead">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label20">
               <property name="text">
                <string>Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelDate">
               <property name="text">
                <string>&lt;&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxResults">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="labelLine">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>1</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>1</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelResults">
               <property name="text">
                <string>&lt;Results&gt;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
